examples/cactus-example-supply-chain-business-logic-plugin/src/main/typescript/business-logic-plugin/supply-chain-cactus-plugin.ts

```diff
@@ -88,7 +109,8 @@ export class SupplyChainCactusPlugin
       contractAddress: this.options.contracts.bookshelfRepository.address,
       contractAbi: this.options.contracts.bookshelfRepository.abi,
       besuApi: this.options.besuApiClient,
-      web3SigningCredential: this.options.web3SigningCredential,
+      web3SigningCredential: this.options
```

Comment:
@jordigiam I'm guessing the casting here was necessary because the options object has the `web3SigningCredential` property declared as optional. Could it be that it's more fitting to change that from optional to mandatory and then no cast is necessary (plus the caller who passes in the options object is informed that way that we do assume the credentials to be present)